src/components/ProjectsList/ProjectList.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
Inconsistent naming of component and folder
Project**s**List (plural) vs ProjectList.js